src/RedisCache/RedisCache/help/Set-AzRedisCache.md

```diff
@@ -51,6 +51,195 @@ PS C:\>Set-AzRedisCache -ResourceGroupName "MyGroup" -Name "MyCache" -RedisConfi
+          EnableNonSslPort   : False
+          RedisVersion       : 4.0.14
+          Size               : 6GB
+          Sku                : Premium
+          Tag                : {}
+          Zone               : []
+``` 
+This command helps in changing persistence method.
+
+
+### Example 6: Scale an Azure Cache for Redis Instance
+
+```
+PS C:\>Set-AzRedisCache -Name "MyCache" -Size "P2" -Sku "Premium"
+
+          PrimaryKey         : pJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          SecondaryKey       : sJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          ResourceGroupName  : MyGroup
+          Id                 : /subscriptions/a559b6fd-3a84-40bb-a450-b0db5ed37dfe/resourceGroups/mygroup/providers/Microsoft.Cache/Redis/MyCache
+          Location           : Central US
+          Name               : mycache
+          Type               : Microsoft.Cache/Redis
+          HostName           : mycache.redis.cache.windows.net
+          Port               : 6379
+          ProvisioningState  : Scaling
+          SslPort            : 6380
+          RedisConfiguration : {[maxmemory-policy, allkeys-random], [maxclients, 7500], [maxmemory-reserved, 200],
+                                [maxfragmentationmemory-reserved, 300]....} 
+          EnableNonSslPort   : False
+          RedisVersion       : 4.0.14
+          Size               : 6GB
+          Sku                : Premium
+          Tag                : {}
+          Zone               : []
+```
+This command increases or decreases the memory size of your redis instance. This will work for all type of caches (Basic, Standard or Premium)
+
+```
+PS C:\>Set-AzRedisCache -Name "MyCache" -Size "P1" -Sku "Premium" 
+
+          PrimaryKey         : pJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          SecondaryKey       : sJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          ResourceGroupName  : MyGroup
+          Id                 : /subscriptions/a559b6fd-3a84-40bb-a450-b0db5ed37dfe/resourceGroups/mygroup/providers/Microsoft.Cache/Redis/MyCache
+          Location           : Central US
+          Name               : mycache
+          Type               : Microsoft.Cache/Redis
+          HostName           : mycache.redis.cache.windows.net
+          Port               : 6379
+          ProvisioningState  : Scaling
+          SslPort            : 6380
+          RedisConfiguration : {[maxmemory-policy, allkeys-random], [maxclients, 7500], [maxmemory-reserved, 200],
+                                [maxfragmentationmemory-reserved, 300]....} 
+          EnableNonSslPort   : False
+          RedisVersion       : 4.0.14
+          Size               : 1GB
+          Sku                : Standard
+          Tag                : {}
+          Zone               : []
+```
+This command will help you in changing tier of your cache (Basic to Standard or Standard to Premium).
```

Comment:
we can also change from Basic to Premium right?

Reply:
```suggestion
This command helps you change the tier of your cache. You can change from Basic to Standard, or Standard to Premium.
```